nixos/modules/services/monitoring/prometheus/exporters/influxdb.nix

```diff
@@ -29,6 +29,8 @@ in
         --web.listen-address ${cfg.listenAddress}:${toString cfg.port} \
         --influxdb.sample-expiry ${cfg.sampleExpiry} ${concatStringsSep " " cfg.extraFlags}
       '';
+      # Need to write to disk
+      ProtectSystem = true;
```

Comment:
What was your reasoning to drop this altogether?

Reply:
When I initially added `ProtectSystem = "strict";` for all exporter, this exporter broke. `ProtectSystem = "full";` was also broken. I tested it this morning and now it seems to work with `ProtectSystem = "strict";`...
I believe the test failed yesterday because my laptop was lagging too much, if I encounter this problem again I might try to add a sleep in the test.